tools/protein_properties/test-data/protein_properties_result1.bed

```diff
@@ -0,0 +1,3 @@
+ID	Sequence	MW	IP	gravy	length	instability	monoisotpoic
+sp|Q9LHB9|PER32_ARATH	MNFSYSSLSTWTTLMTLGCLLLHSSISSAQLTPTFYDNTCPSVFTIVRDTIVNELRSDPRIAASILRLHFHDCFVNGCDASILLDNTTSFRTEKDAAPNANSARGFPVIDRMKAAVETACPRTVSCADILTIAAQQAVNLAGGPSWRVPLGRRDSLQAFFALANTNLPAPFFTLPQLKASFQNVGLDRPSDLVALSGGHTFGKNQCQFIMDRLYNFSNTGLPDPTLNTTYLQTLRGQCPRNGNQTVLVDFDLRTPTVFDNKYYVNLKELKGLIQTDQELFSSPNATDTIPLVREYADGTQKFFNAFVEAMNRMGNITPLTGTQGQIRQNCRVVNSNSLLHDVVEIVDFVSSM	38846.6361	6.20635986328	-0.0943181818182	352	31.0301420455	0.0880681818182
```

Comment:
Yeah thought about this too, but then it's very easy to get the FASTA file back (for example after filtering) with tabular-to-faster.
On the other hand, the user could extract al sequences from the original FASTA with the filtered ID's.
To make it short I'm not sure :) Moving it as last column? Or removing it?